deploy/README-multicloud.md

```diff
@@ -0,0 +1,100 @@
+# Configuring Service Assurance Framework for Multiple Clouds
+
+Multiple Openstack clouds can be configured to target a single instance of SAF.
+There are a few steps to get this set up:
+
+1. Plan the AMQP address prefixes to use for each cloud
+1. Deploy metrics and notification SmartGateways for each cloud to listen on the
+  corresponding address prefixes
+1. Configure each cloud to send it's metrics and notifications to SAF on the
```

Comment:
s/notifications/events

Only suggesting this change since these are the terms we've been using.